apps/synapse-portal-framework/src/components/DetailsPage/DetailsPage.test.tsx

```diff
@@ -1,310 +1,9 @@
-        tabLayout: [
-          {
-            title: 'Subtab 1',
-            uriValue: 'Subtab1',
-            synapseConfigArray: [
-              {
-                name: 'Markdown',
-                standalone: true,
-                props: {
-                  markdown: 'Synapse component in first subtab',
-                },
-              },
-            ],
-          },
-          {
-            title: 'Subtab 2',
-            uriValue: 'Subtab2',
-            synapseConfigArray: [
-              {
-                name: 'Markdown',
-                standalone: true,
-                props: {
-                  markdown: 'Synapse component in second subtab',
-                },
-              },
-            ],
-          },
-        ],
-      },
-      {
-        title: 'Tab 2',
-        uriValue: 'Tab2',
-        synapseConfigArray: [
-          {
-            name: 'Markdown',
-            standalone: true,
-            props: {
-              markdown: 'Synapse component in second tab',
-            },
-          },
-        ],
-      },
-    ]
-    renderWithContext(<DetailsPage tabLayout={tabLayout} sql="" />)
-
-    const getTab1 = async () => await screen.findByText('Tab 1')
-    const getTab2 = async () => await screen.findByText('Tab 2')
-    const getSubtab1 = async () => await screen.findByText('Subtab 1')
-    const getSubtab2 = async () => await screen.findByText('Subtab 2')
-
-    // Tab buttons should be visible
-    await getTab1()
-    await getTab2()
-    await getSubtab1()
-    await getSubtab2()
-
-    // Component in first subtab should be visible, second should not
-    await screen.findByText('Synapse component in first subtab')
-    expect(screen.queryByText('Synapse component in second subtab')).toBeNull()
-
-    // Click the second subtab and which component is visible should switch
-    await userEvent.click(await getSubtab2())
-
-    await screen.findByText('Synapse component in second subtab')
-    expect(screen.queryByText('Synapse component in first subtab')).toBeNull()
-
-    // Click the first subtab and it should switch back
-    await userEvent.click(await getSubtab1())
-
-    await screen.findByText('Synapse component in first subtab')
-    expect(screen.queryByText('Synapse component in second subtab')).toBeNull()
-
-    // Click tab 2 and the subtabs and content should disappear
-    await userEvent.click(await getTab2())
-
-    await screen.findByText('Synapse component in second tab')
-    expect(screen.queryByText('Subtab 1')).toBeNull()
-    expect(screen.queryByText('Subtab 2')).toBeNull()
-    expect(screen.queryByText('Synapse component in first subtab')).toBeNull()
-    expect(screen.queryByText('Synapse component in second subtab')).toBeNull()
-
-    // Click tab 1 and the subtabs should reappear
-    await userEvent.click(await getTab1())
-    await screen.findByText('Subtab 1')
-    await screen.findByText('Subtab 2')
-  })
-
-  it('Test overrideSqlSourceTable', async () => {
```

Comment:
Note that props are no longer injected into components, so even these old tests that capture the 'spirit' of the details page don't make sense to preserve.